scss/layout/_section.scss

```diff
@@ -13,55 +15,59 @@
-    $fd-section-padding--bottom: fd-space("reg") !default;
-    $fd-section-header-margin--bottom: fd-space("reg") !default;
-    $fd-section-title-color: fd-color("text",2) !default;
-    --fd-section-title-color: var(--fd-color-text-2);
+  $fd-section-padding--top: fd-space(s) !default;
+  $fd-section-padding--bottom: fd-space("reg") !default;
+  $fd-section-header-margin--bottom: fd-space("reg") !default;
+  $fd-section-title-color: fd-color("text", 2) !default;
+  --fd-section-title-color: var(--fd-color-text-2);
 
-    @include fd-clearfix();
-    padding: $fd-section-padding--top $fd-section-padding $fd-section-padding--bottom;
-    padding: $fd-section-padding--top var(--fd-section-padding-x) $fd-section-padding--bottom;
-    border-bottom: none;
+  @include fd-reset();
+  @include fd-clearfix();
+  padding: $fd-section-padding--top $fd-section-padding $fd-section-padding--bottom;
+  padding: $fd-section-padding--top var(--fd-section-padding-x) $fd-section-padding--bottom;
+  border-bottom: none;
 
-    background-color: var(--fd-section-background-color, transparent);
+  background-color: var(--fd-section-background-color, transparent);
 
-    &:last-child,
-    &--no-border {
-        border-bottom: 0;
-    }
-    &--full-bleed,
-    &--bleed {
-        padding-right: 0;
-        padding-left: 0;
-        .#{$block}__header, .#{$block}__footer {
-            margin-left: $fd-section-padding;
-            margin-left: var(--fd-section-padding-x);
-            margin-right: $fd-section-padding;
-            margin-right: var(--fd-section-padding-x);
-        }
-    }
-    &__header {
-        min-height: fd-space(10);
-        display: flex;
-        align-items: center;
-        margin-bottom: $fd-section-header-margin--bottom;
-    }
-    &__title {
-        @include fd-type("4");
-        @include fd-var-color("color", $fd-section-title-color, --fd-section-title-color);
-        flex: 1;
-        margin-bottom: 0;
-    }
-    &__actions {
-        margin-left: auto;
-    }
-    &__footer {
-        display: flex;
-        justify-content: center;
+  &:last-child,
+  &--no-border {
+    border-bottom: 0;
+  }
+  &--full-bleed,
+  &--bleed {
+    @include fd-reset();
+    padding-right: 0;
+    padding-left: 0;
+    .#{$block}__header,
+    .#{$block}__footer {
+      margin-left: $fd-section-padding;
+      margin-left: var(--fd-section-padding-x);
+      margin-right: $fd-section-padding;
+      margin-right: var(--fd-section-padding-x);
     }
+  }
+  &__header {
+    @include fd-reset();
+    min-height: fd-space(10);
+    display: flex;
+    align-items: center;
+    margin-bottom: $fd-section-header-margin--bottom;
+  }
+  &__title {
+    @include fd-reset();
+    @include fd-type("4");
+    @include fd-var-color("color", $fd-section-title-color, --fd-section-title-color);
+    flex: 1;
+    margin-bottom: 0;
+  }
+  &__actions {
+    margin-left: auto;
+  }
+  &__footer {
+    display: flex;
+    justify-content: center;
+  }
```

Comment:
It's tough to tell if anything actually changed in this file with the reformatting.  Were there any changes?

Reply:
I reverted the reformatted code